src/passport-saml/saml.ts

```diff
@@ -473,14 +499,14 @@ class SAML {
 
     overrideParams = overrideParams || {};
     Object.keys(overrideParams).forEach(function(k) {
-      additionalParams[k] = overrideParams[k];
+      additionalParams[k] = overrideParams![k];
```

Comment:
I'm surprised that you need the `!` here at all. I expect that TypeScript will note that all values of `k` exist because of the iterator. What am I missing?